contracts/MerkleDistributorWithDeadline.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.15;
+
+import "./MerkleDistributor.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import {IERC20, SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract MerkleDistributorWithDeadline is MerkleDistributor, Ownable {
+    using SafeERC20 for IERC20;
+
+    uint256 public immutable endTime;
```

Comment:
should the owner be able to extend endTime?

Reply:
> should the owner be able to extend endTime?

The product people responsible for designing the airdrop did not mention this, so I assumed no.

Reply:
But if they say we dont need it then happy to leave it as-is!

Reply:
I can confirm the answer is no.